Makefile

```diff
@@ -51,6 +47,14 @@ ifeq ($(LEDGER_ENABLED),true)
   endif
 endif
 
+### Update Build flags
+BUILD_FLAGS = -tags "${BUILD_TAGS}" -ldflags \
```

Comment:
...then append the tags here:

```Makefile
BUILD_FLAGS += -tags "$(BUILD_TAGS)"
```

So that build flags that are not affected by this change will be declared among the other variables.